cg/meta/workflow/mutant.py

```diff
@@ -105,6 +105,27 @@ def get_sample_parameters(self, sample_obj: models.Sample) -> MutantSampleConfig
             ),
         )
 
+    def get_internal_NTCs(self, case_id: str) -> List[models.Sample]:
+        """Retrieve a list of lims ids for negative controls created by Clinical Genomics"""
+        case_obj = self.status_db.family(case_id)
+        samples: List[models.Sample] = [link.sample for link in case_obj.links]
+        ticket_id: int = samples[0].ticket_number
+        # 1. ticket_id will lead to samples that all come from the same pool
+        # 2. In the pool from "1." there will be samples from another ticket than ticket_id given as input
+        # 3. Return a list of sample objects from "2."
+
+    def check_if_NTCs_are_clean(self, case_id: str) -> bool:
```

Comment:
```suggestion
    def are_all_negative_controls_clean(self, case_id: str) -> bool:
```
I think it might help readability if bool methods are named something that makes it clear it is a get method (returns something) and that it returns something boolean like "is/are/has" and that the abbreviation is written out

Reply:
updated this, also changed where this function is used